python/src/equistore/operations/lstsq.py

```diff
@@ -2,6 +2,7 @@
 
 import numpy as np
 
+from ..labels import Labels
```

Comment:
speaking with guillaume the idea is to put the imports in the example in the pydocs

Reply:
Thanks for the suggestions!

For the linter issue: how do we want to fix then? With # noqa so that the linter ignores the line or with the imports directly in the example? If the second: should we then import everything that's needed to reproduce the example with copy & paste?

I can also comment on all the points suggested by Alex.

Reply:
> For the linter issue: how do we want to fix then? With # noqa so that the linter ignores the line or with the imports directly in the example? If the second: should we then import everything that's needed to reproduce the example with copy & paste?

We decided to include the import in the docstring. I added a suggestion where it should be added. That should work.

Reply:
> I can also comment on all the points suggested by Alex.

Thanks, would be very helpful!